src/platform/site-wide/sass/merger.scss

```diff
@@ -372,11 +372,6 @@
     }
   }
 
-  #preview-site-alert {
-    color: $color-black;
-    position: relative;
-  }
```

Comment:
Not sure why this was `position: relative`. Are there any elements ever that this banner should take precedence over? It doesn't seem like it.

CC: @jbalboni  @RobAWilkinson @accbjt